### PR TITLE
[skia] Remove Options Files with Only max_len Flag

### PR DESCRIPTION
**Pull request by werdnaH**
@kjlubick @jonathanmetzman 
Removed all options files with only max_len and add if statement Fuzz cpps to check the max length.

### CODE REVIEW

**Comment by kjlubick** [comment]
Please rebase and upload a CL to skia to address options first.

**Comment by inferno-chromium** [comment]
@werdnaH - ping us on chat when skia master has the length checks in fuzz targets, then we can merge this PR, or leave a comment here.

File: projects/skia/Dockerfile [hunk 1]
```diff
@@ -80,38 +80,8 @@ COPY build.sh $SRC/
 COPY skia.diff $SRC/skia/skia.diff
 RUN git apply skia.diff
 
-COPY region_deserialize.options $SRC/skia/region_deserialize.options
-COPY region_set_path.options $SRC/skia/region_set_path.options
 COPY image_filter_deserialize.options $SRC/skia/image_filter_deserialize.options
```
Comment (inferno-chromium) [inline projects/skia/Dockerfile:83]: @kjlubick - do these two image_filter_deserialize*.options have to stay ?

Comment (kjlubick) [reply projects/skia/Dockerfile:83]: I vaguely remember us experimenting with two different approaches to fuzzing image_filter_deserialize. Looking at the fuzzer history, it seems the different options *do* have an impact on performance. I think it's worth keeping them in for now until we can maybe do a little more investigation.

Comment (werdnaH) [reply projects/skia/Dockerfile:83]: All the options files are deleted except for image_filter_deserialize_width.options which has use_counter=0

**Comment by werdnaH** [comment]
All the options files are deleted except for image_filter_deserialize_width.options which has use_counter=0

**Pull request merged**